llvm/include/llvm/IR/DiagnosticInfo.h

```diff
@@ -712,16 +711,16 @@ class DiagnosticInfoIROptimization : public DiagnosticInfoOptimizationBase {
     *this << Msg.str();
   }
 
-  const Value *getCodeRegion() const { return CodeRegion; }
+  const BasicBlock *getCodeRegion() const { return CodeRegion; }
 
   static bool classof(const DiagnosticInfo *DI) {
     return DI->getKind() >= DK_FirstRemark && DI->getKind() <= DK_LastRemark;
   }
 
 private:
-  /// The IR value (currently basic block) that the optimization operates on.
+  /// The IR value that the optimization operates on.
```

Comment:
```suggestion
  /// The IR region (currently basic block) that the optimization operates on.
```

Should not refer to IR value now that it is BasicBlock *